wire-ios/Wire-iOS/Sources/UserInterface/StartUI/StartUI/StartUIViewController.swift

```diff
@@ -312,13 +321,10 @@ final class StartUIViewController: UIViewController {
     /// - API >= v8
     /// https://wearezeta.atlassian.net/wiki/spaces/ENGINEERIN/pages/1712979983/Channels
 
-    private var canCreateChannel: Bool {
+    private var areChannelsSupported: Bool {
         guard let backendInfoApiVersion = BackendInfo.apiVersion else {
             return false
         }
-        guard userSession.channelsFeature.canCreateChannels(role: userSession.selfUser.teamRole) else {
-            return false
-        }
```

Comment:
I'm not sure if the conditions are correct now, however, I needed to split it up:
1. I assume the create-channel button needs to be visible when the backend supports it.
2. We now also show it even if the account is not a team account.